From ce7d724a2d563bbb1634901b1ece8321abf86342 Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Sat, 28 Apr 2012 17:04:45 -0300
Subject: [PATCH 18/40] 4.2.8 Hijos de Hodir: El Fuego del Refinador

Deleted conditions for slag covered metal loot, players should be able to loot it anytime
---
 .../2012_04_28_100_world_the_refiners_fire.sql     |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_28_100_world_the_refiners_fire.sql

diff --git a/sql/updates/world/reanemu/2012_04_28_100_world_the_refiners_fire.sql b/sql/updates/world/reanemu/2012_04_28_100_world_the_refiners_fire.sql
new file mode 100644
index 0000000..76e666e
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_28_100_world_the_refiners_fire.sql
@@ -0,0 +1 @@
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=1 AND `SourceGroup`=29375 AND `SourceEntry`=41556;
-- 
1.7.8.msysgit.0

